### PR TITLE
VRPB instances

### PR DESCRIPTION
**Pull request by leonlan**
This PR adds the largest 90 VRPB instances and best-known values from Quieroga et al. (2020).

Part of https://github.com/PyVRP/PyVRP/issues/434.

### CODE REVIEW

**Comment by leonlan** [comment]
The benchmarking instructions will be as follows:

```
* For the VRPB, we use `the 90 largest VRPB instances of Queiroga et al. (2020) <https://github.com/PyVRP/Instances/tree/main/VRPB>`_.
  These instances range in size from 772 to 1000 customers.
  Each instance has :math:`n` clients; the runtimes are computed as :math:`2.4 n` seconds on a reference CPU with PassMark score 2000.
  Thus, an instance with 100 clients is ran for 240 seconds, assuming a CPU with PassMark score 2000.
  For this benchmark, PyVRP needs to be compiled with the CVRP problem setting, and the ``round`` rounding function should be used.
  Additionally, the configuration specified in ``configs/cvrp.toml`` should be used.
```

File: VRPB/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# VRPB
+
+VRP with backhaul instances of Queiroga et al. (2020).
+Downloaded from [VRP-REP](http://www.vrp-rep.org/references/item/queiroga-et-al-2019.html) and modified to fit the VRPLIB format. 
+This repository only includes the 90 largest VRPB instances, which we use for benchmarking PyVRP.
+The other instances can be found in [this zip](https://github.com/PyVRP/PyVRP/files/14402701/VRPB.zip).
```
Comment (N-Wouda) [inline VRPB/README.md]: Which variants? I see three `n1001`, but what's the difference?

Comment (N-Wouda) [reply VRPB/README.md]: Maybe say something about "there are three variants of the VRPB problem: <list>. We include the 30 largest instances of Queiroga et al. (2020) of each variant in this repository."

File: VRPB/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+### References
+
+Queiroga, E., Frota, Y., Sadykov, R., Subramanian, A., Uchoa, E., & Vidal, T. (2020). On the exact solution of vehicle routing problems with backhauls. European Journal of Operational Research, 287(1), 76–89. https://doi.org/10.1016/j.ejor.2020.04.047
```
Comment (N-Wouda) [inline VRPB/README.md]:
```suggestion
Queiroga, E. et al. (2020). On the exact solution of vehicle routing problems with backhauls. European Journal of Operational Research, 287(1): 76–89. https://doi.org/10.1016/j.ejor.2020.04.047.
```

Comment (N-Wouda) [reply VRPB/README.md]: (this is in line with the other directories)

File: VRPB/X-n1001-50-k22.sol [hunk 3]
```diff
@@ -0,0 +1,2 @@
+Cost: 49978
+Comment: Best known value from Quieroga et al. (2020) 'On the exact solution of vehicle routing problems with backhauls.'
```
Comment (N-Wouda) [inline VRPB/X-n1001-50-k22.sol:2]: Question: are these known optimal, or BKS?

**Comment by N-Wouda** [comment]
> [...] These instances range in size from 772 to 1000 customers. [...]

I'd also put this in the readme here - it's useful to have a rough indication of size.

**Comment by N-Wouda** [comment]
LGTM with a few minor comments!

File: VRPB/X-n1001-50-k22.sol (hunk 3 above)
Comment (leonlan) [reply VRPB/X-n1001-50-k22.sol:2]: These solutions are only BKS

**Pull request merged**